src/assets/translations/fr/main.json

```diff
@@ -2315,5 +2330,12 @@
         "body": "Bien que les pools de liquidités offrent des opportunités de gains, il est important d'être conscient des risques impliqués, notamment des pertes impermanentes, des vulnérabilités des contrats intelligents et de la volatilité du marché. Les utilisateurs doivent effectuer des recherches approfondies avant de participer."
       }
     }
+  },
+  "watchlist": {
+    "empty": {
+      "title": "Commencez à créer votre liste de suivi",
+      "body": "Partout où vous voyez l’icône étoile, vous pouvez l’utiliser pour ajouter des actifs ici.",
```

Comment:
Nitpick but `Vous pouvez l’utiliser pour ajouter des actifs ici partout où vous voyez l’icône étoile.` sounds more natural to me

Reply:
Sounds much better indeed @firebomb1 !

Reply:
```suggestion
      "body": "Vous pouvez utiliser l'icône étoile partout où vous la voyez afin d'ajouter des actifs ici.",
```